hw/ip/sysrst_ctrl/doc/dv/index.md

```diff
@@ -30,15 +30,15 @@ For detailed information on SYSRST_CTRL design features, please see the [SYSRST_
 SYSRST_CTRL testbench has been constructed based on the [CIP testbench architecture]({{< relref "hw/dv/sv/cip_lib/doc" >}}).
 
 ### Block diagram
-![Block diagram](tb.svg)
+![Block diagram](sysrst_ctrl_tb_block_diagram.svg)
 
 ### Top level testbench
 The top level testbench is located at `hw/ip/sysrst_ctrl/dv/tb.sv`.
 It instantiates the SYSRST_CTRL DUT module `hw/ip/sysrst_ctrl/rtl/sysrst_ctrl.sv`.
 In addition, it instantiates the following interfaces, connects them to the DUT and sets their handle into `uvm_config_db`:
 * [Clock and reset interface]({{< relref "hw/dv/sv/common_ifs" >}})
 * [TileLink host interface]({{< relref "hw/dv/sv/tl_agent/README.md" >}})
-* SYSRST_CTRL IOs
+* SYSRST_CTRL IOs ({{< relref "hw/ip/sysrst_ctrl/env/sysrst_ctrl_if" >}}))
```

Comment:
Missing dv in ...ctrl/dv/env... Also, this link may fail since it won't find an .md file. You may mean a link to the source file, which is through github.